docs/artifacts/three-sources-of-truth.html

```
@@ -0,0 +1,333 @@
+<title>Three Sources Of Truth</title>
```

HTMLHint (1.9.2)

[error] Doctype must be declared before any non-comment content.

(doctype-first)